packages/webapp/src/routes/credits-purchase/credits-development-location.js

```diff
@@ -0,0 +1,46 @@
+import path from 'path'
+import { deleteBlobFromContainers } from '../../utils/azure-storage.js'
+import constants from '../../utils/constants.js'
+
+const href = '#dev-details-checked-yes'
+const handlers = {
+  get: (request, h) => {
+    const context = getContext(request)
+    return h.view(constants.views.CREDITS_CONFIRM_DEV_DETAILS, context)
+  },
+  post: async (request, h) => {
+    const confirmDevDetails = request.payload.confirmDevDetails
+    const metricUploadLocation = request.yar.get(constants.redisKeys.CREDITS_METRIC_LOCATION)
+    request.yar.set(constants.redisKeys.METRIC_FILE_CHECKED, confirmDevDetails)
+    if (confirmDevDetails === constants.creditsCheckDetails.NO) {
+      await deleteBlobFromContainers(metricUploadLocation)
+      request.yar.clear(constants.redisKeys.CREDITS_METRIC_LOCATION)
+      return h.redirect(constants.routes.CREDITS_UPLOAD_METRIC)
+    } else if (confirmDevDetails === constants.creditsCheckDetails.YES) {
+      return h.redirect('#')
```

Comment:
is this redirect correct?

Reply:
Further page is not developed and when this screen was developed at that time complete prototype was not available so added redirecting to "#"

Reply:
Could you please check the following change made? 

https://github.com/DEFRA/biodiversity-net-gain-service/pull/618/commits/4eba8a811e1773828a18cb4f44e8f5a185911031

Reply:
Yes. The actual page is not developed yet and when this PR was created any further prototype pages was not there